apps/explorer/src/pages/AddressInfoPage/index.tsx

```diff
@@ -102,12 +109,18 @@ const AddressInfoPage = () => {
     lastKnownMempoolTxs.current = addressMempoolTransactions
   }, [addressMempoolTransactions, refetchTxList])
 
+  const tokensPrices = useTokensPrices([ALPH.symbol, ...fungibleTokensMetadata.map((t) => t.symbol)])
+
   if (!addressHash) {
     displaySnackbar({ text: t('The address format seems invalid'), type: 'alert' })
-    navigate('404')
+    navigate('/404')
+    return null
   }
 
-  const tokensPrices = useTokensPrices([ALPH.symbol, ...fungibleTokensMetadata.map((t) => t.symbol)])
+  const addressWithoutGroup = addressHash.split(':')[0]
```

Comment:
nit: I saw you used this syntax before and I really like it :)

```suggestion
  const [addressWithoutGroup] = addressHash.split(':')
```

Reply:
I updated this in my PR